sdk/src/Services/S3/Custom/Model/GetObjectResponse.cs

```diff
@@ -1042,5 +1043,10 @@ internal WriteObjectProgressArgs(string bucketName, string key, string filePath,
         /// True if writing is complete
         /// </summary>
         public bool IsCompleted { get; private set; }
+
+        /// <summary>
+        /// The original TransferUtilityDownloadRequest created by the user.
+        /// </summary>
+        public TransferUtilityDownloadRequest Request { get; internal set; }
```

Comment:
why does this need to be here? what would this look like when this operation is generated? and why is it not internal at the very least?